src/models/inclusion_proof.rs

```diff
@@ -28,7 +25,12 @@ pub struct InclusionProof {
 }
 
 impl InclusionProof {
-    pub fn new(log_index: i32, root_hash: String, tree_size: i32, hashes: Vec<String>) -> InclusionProof {
+    pub fn new(
+        log_index: i32,
```

Comment:
I see this one as `i64` too
https://github.com/sigstore/rekor/blob/main/pkg/sharding/log_index.go#L28